template.yml

```diff
@@ -2,63 +2,24 @@ AWSTemplateFormatVersion: '2010-09-09'
 Transform: 'AWS::Serverless-2016-10-31'
```

Comment:
We don't need this file do we? It's for AWS SAM, but we are using serverless.

Reply:
I wasn't sure whether it would be used by some of the webpack stuff we'd be using later for packaging the individual lambda. But if it's for SAM I'll delete

Reply:
Can always put it back if we do need it, but hopefully we won't need two configuration files.